frontend/src/features/workspace/mutations.ts

```diff
@@ -151,3 +153,15 @@ export const useDeleteFormMutation = () => {
 
   return { deleteFormMutation }
 }
+
+export const useAdminFeedbackMutation = () => {
+  const createAdminFeedbackMutation = useMutation((rating: number) =>
```

Comment:
Again, can be more specific in typing!

```suggestion
  const createAdminFeedbackMutation = useMutation((rating: AdminFeedbackRatingUp | AdminFeedbackRatingDown) =>
```

Reply:
[632f87e](https://github.com/opengovsg/FormSG/pull/6465/commits/632f87e807e3a074d773673b2453226cbf620df1)

Moved it to an enum instead! else its a little ugly hahah 😅 `rating: typeof AdminFeedbackRatingUp | typeof AdminFeedbackRatingDown`